extension/indexer/tx_indexer.go

```diff
@@ -1,116 +0,0 @@
-// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package indexer
```

Comment:
Could we revert this diff? This should make this PR easier to review, and I think the only change needed to this file is the function signature of `Accepted`